.golangci.yml

```diff
@@ -18,7 +18,7 @@ issues:
 
 linters-settings:
   govet:
-    enable=fieldalignment: true
```

Comment:
Moreover, we need to have a new target for the Makefile that we use for Kubebuilder as well.
It can be here, and we do all by once, for our samples and tooling or can be in a follow up PR

Reply:
oh no, I mean disable `fieldalignment` rule

If you check lint job logs, all errors were related to this rule

I made the change because `golangci-lint config verify` complained about this configuration

but we can change this part of project Makefile in a separate PR

Reply:
undid changes to configuration, just added new command to project Makefile, without calling it in CI

Reply:
>I made the change because golangci-lint config verify complained about this configuration


Should it not fail in the CI if the lint raises errors and we revert the change?